File: app/src/main/java/com/halfplatepoha/jisho/jdb/Kanji.java
package com.halfplatepoha.jisho.jdb;

import io.realm.RealmList;
import io.realm.RealmObject;

public class Kanji extends RealmObject {
    
    public RealmList<Codepoint> codepoints;
    public RealmList<DicNumber> dicNumbers;
    public int freq;
    public int grade;
    public String literal;
    public RealmList<Meaning> meanings;
    public RealmList<QueryCode> queryCodes;
    public RealmList<Reading> readings;
    public int strokeCount;

}

File: app/src/main/java/com/halfplatepoha/jisho/v2/realm/Kanji.kt
package com.halfplatepoha.jisho.v2.realm

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Kanji: RealmObject() {

    @PrimaryKey
    var _id: Int = 0

    var onyomi: String? = null
    var kunyomi: String? = null
    var nanori: String? = null
    var radical: Int? = null
    var grade: Int? = null
    var layout: Int? = null
    var skip: String? = null

}